addons/html_editor/static/src/main/font/color_plugin.js

```diff
@@ -74,6 +76,29 @@ export class ColorPlugin extends Plugin {
         );
     }
 
+    normalize(root) {
+        for (const el of selectElements(root, "font")) {
+            const closestEl = closestElement(el.parentElement, "font");
+            if (closestEl) {
+                // Compare non-empty styles with closest element.
+                const stylesMatch = FONT_STYLE_PROPERTIES.every((prop) => {
+                    const elStyle = el.style.getPropertyValue(prop);
+                    const closestElStyle = closestEl.style.getPropertyValue(prop);
+                    return !elStyle || elStyle === closestElStyle;
+                });
+
+                // Check if all classes match the closest element.
+                const classListMatch = [...el.classList].every((cls) =>
+                    closestEl.classList.contains(cls)
+                );
+
+                if (stylesMatch && classListMatch) {
+                    unwrapContents(el);
+                }
+            }
+        }
```

Comment:
```suggestion
            if (closestEl && el.cloneNode(false).isEqualNode(closestEl.cloneNode(false))) {
                // If the font is equal to the closest font, unwrap it.
                unwrapContents(el);
            }
```

Can you test it would work using [isEqualNode](https://developer.mozilla.org/en-US/docs/Web/API/Node/isEqualNode)? Same thing for the code in `font_plugin.js`

Reply:
The isEqualNode method requires an exact match of styles and classes. In this example:
```html
<p>
  <font style="background-color: rgba(255, 0, 0, 0.6);" class="text-o-color-1">
    parent
    <font style="background-color: rgba(255, 0, 0, 0.6);">child</font>
  </font>
</p>
```
Although the inner tag has the same style, it lacks the class, so isEqualNode returns false. For detecting such redundant elements, a custom style comparison is needed.